omero/sysadmins/limitations.txt

```diff
@@ -26,6 +26,13 @@ correctly. OpenJDK and OracleJDK versions 6 and 7 are also supported.
 Windows OS issues
 -----------------
 
+Django 1.8 will not work on Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OMERO.web has been upgraded to use Django 1.8 but support for this on Windows
```

Comment:
I wouldn't add that paragraph at all. This is only IIS issue. We don;t know how many people deploy using Apache.